cmd/docs/generate.go

```diff
@@ -15,16 +15,41 @@
 package main
 
 import (
+	"fmt"
 	"log"
+	"path"
+	"path/filepath"
+	"strings"
 
 	"github.com/banzaicloud/banzai-cli/cmd"
 	"github.com/spf13/cobra/doc"
 )
 
 func main() {
 	cmd.Init("", "", "", "")
+
+	const fmTemplate = `---
+title: %s
+slug: %s
+---
+
+> This file was generated automatically. Do not modify it.
```

Comment:
why is this needed? do we want it to be shown in the rendered docs?

Reply:
Yes, so noone starts editing something in the md file, because it will be automatically lost

Reply:
but how is this information relevant for the readers of the rendered site?